runtime/caml/fiber.h

```diff
@@ -195,13 +195,14 @@ extern value caml_global_data;
 #define Trap_link(tp) ((tp)[1])
 
 struct stack_info** caml_alloc_stack_cache (void);
-CAMLextern struct stack_info* caml_alloc_main_stack (uintnat init_size);
+CAMLextern struct stack_info* caml_alloc_main_stack (uintnat init_wsize);
 void caml_scan_stack(scanning_action f, void* fdata,
                      struct stack_info* stack, value* v_gc_regs);
 /* try to grow the stack until at least required_size words are available.
    returns nonzero on success */
-int caml_try_realloc_stack (asize_t required_size);
-void caml_change_max_stack_size (uintnat new_max_size);
+int caml_try_realloc_stack (asize_t required_wsize);
+uintnat caml_get_init_stack_wsize(void);
```

Comment:
@dra27 do these functions need CAMLextern annotations?

Reply:
Yes, please!

Reply:
Good catch, I added the annotations.